backend/accounts_test.go

```diff
@@ -21,11 +21,21 @@ import (
 	"github.com/btcsuite/btcutil/hdkeychain"
 	coinpkg "github.com/digitalbitbox/bitbox-wallet-app/backend/coins/coin"
 	"github.com/digitalbitbox/bitbox-wallet-app/backend/config"
+	keystoremock "github.com/digitalbitbox/bitbox-wallet-app/backend/keystore/mocks"
 	"github.com/digitalbitbox/bitbox-wallet-app/backend/signing"
+	"github.com/digitalbitbox/bitbox-wallet-app/util/errp"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func mustKeypath(keypath string) signing.AbsoluteKeypath {
+	kp, err := signing.NewAbsoluteKeypath(keypath)
+	if err != nil {
+		panic(err)
```

Comment:
A panic prevents all following tests from running, apart from avoiding an ugly stacktrace. A better way imho would be to fail only one specific test where this is used. for example,

```go
func makeKeypath(t *testing.T, p string) signing.AbsoluteKeypath {
  t.Helper()
  ...
  if err != nil {
    t.Fatalf("%s: %v", p, err)
  }
  return kp
}
```